### PR TITLE
Do not perform reality check on accounts that does not apply

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/javascript/binary/static/reality-check.js [hunk 1]
```diff
@@ -130,6 +135,10 @@ var RealityCheck = (function() {
     }
 
     function init() {
+        if (!page.client.require_reality_check()) {
```
Comment (binary-hamedanchi) [inline src/javascript/binary/static/reality-check.js]: It is not guaranteed to get the websocket response before `Page.on_load()` which is calling this `init()`. e.g. right after login or switch loginids. Therefore, at those moments, the `has_reality_check` value would not be there, yet.

If so, it's probably needed to call `RealityCheck.init()` after receiving the websocket response in `page.client. response_landing_company_details()`.

Comment (qingweibinary) [inline src/javascript/binary/static/reality-check.js]: You're right ~ 
I'll fix it ~

File: src/javascript/binary/base/page.js [hunk 2]
```diff
@@ -185,7 +188,12 @@ Client.prototype = {
 
         // allowed markets
         if(this.is_logged_in) {
-            if(!TUser.get().is_virtual && !this.get_storage_value('allowed_markets') && TUser.get().landing_company_name) {
+            if(
+                !TUser.get().is_virtual &&
```
Comment (negar-binary) [inline src/javascript/binary/base/page.js]: can you check if we can use @binary-hamedanchi's new is_virtual() check instead of calling TUser?

Comment (qingweibinary) [inline src/javascript/binary/base/page.js]: I think so ~

**Pull request merged**